jetty-core/jetty-websocket/jetty-websocket-jetty-common/src/main/java/org/eclipse/jetty/websocket/common/JettyWebSocketFrameHandler.java

```diff
@@ -145,15 +145,16 @@ public void onOpen(CoreSession coreSession, Callback callback)
                 container.notifySessionListeners((listener) -> listener.onWebSocketSessionOpened(session));
 
             callback.succeeded();
+
+            if (openHandle != null)
+                autoDemand();
+            else
+                session.getCoreSession().demand();
```

Comment:
Please refactor this into a method called `internalDemand()`, so it's clearer what it does (and it can be javadoc-ed).

Reply:
Done.